include/benchmark/reporter.h

```diff
@@ -105,12 +110,45 @@ class BenchmarkReporter {
   // reported.
   virtual void Finalize();
 
+  // REQUIRES: 'out' shall be valid for the lifetime of the reporter.
+  // Set the stream used for non-error output to the value specified by 'out'.
+  void SetOutputStream(std::ostream& out) {
```

Comment:
doesn't seem to, just arguments.

ftr, i don't like this rule, but there it is.